modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/application/authz/ApplicationAuthzTenantTestCase.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+package org.wso2.identity.integration.test.application.authz;
+
+import org.apache.axis2.context.ConfigurationContext;
+import org.apache.axis2.context.ConfigurationContextFactory;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.http.HttpResponse;
+import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.util.EntityUtils;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.identity.integration.common.clients.application.mgt.ApplicationManagementServiceClient;
+import org.wso2.identity.integration.common.clients.entitlement.EntitlementPolicyServiceClient;
+import org.wso2.identity.integration.common.clients.sso.saml.SAMLSSOConfigServiceClient;
+import org.wso2.identity.integration.common.clients.usermgt.remote.RemoteUserStoreManagerServiceClient;
+import org.wso2.identity.integration.test.util.Utils;
+import org.wso2.identity.integration.test.utils.CommonConstants;
+
+import java.io.File;
+import java.net.URL;
+
+public class ApplicationAuthzTenantTestCase extends ApplicationAuthzTestCase {
+
+    private static final String AZ_TEST_TENANT_ROLE = "azTestTenantRole";
+    private static final String HTTP_REDIRECT = "HTTP-Redirect";
+    private static final String AZ_TEST_TENANT_USER = "azTestTenantUser";
+    private static final String AZ_TEST_TENANT_USER_PW = "azTest123";
+    private static final String NON_AZ_TEST_TENANT_USER = "nonAzTestTenantUser";
+    private static final String NON_AZ_TEST_TENANT_USER_PW = "nonAzTest123";
+    private static final String WSO2_DOMAIN = "@wso2.com";
+    private static final Log log = LogFactory.getLog(ApplicationAuthzTenantTestCase.class);
+    // SAML Application attributes
+    private static final String USER_AGENT = "Apache-HttpClient/4.2.5 (java 1.5)";
+    private static final String APPLICATION_NAME = "travelocity.com-saml-tenantwithoutsigning";
+    private static final String ACS_URL = "http://localhost:8490/%s/home.jsp";
```

Comment:
yeah, it need to be read from common constants, thats how it has been done in test cases which use travelocity and playground app, otherwise it can be really difficult to change the ports if any requirement  as in https://github.com/wso2/product-is/pull/199 happens